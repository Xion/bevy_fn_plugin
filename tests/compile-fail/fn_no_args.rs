#[macro_use] extern crate bevy_fn_plugin;

#[bevy_plugin]
fn no_arg_fn() {}

fn main() {}
