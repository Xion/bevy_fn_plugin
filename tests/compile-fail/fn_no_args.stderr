error: Invalid plugin function signature
 --> tests/compile-fail/fn_no_args.rs:4:1
  |
4 | fn no_arg_fn() {}
  | ^^^^^^^^^^^^^^
  |
  = help: Function should take a single argument of the type `&mut bevy::app::App`
